src/explore-education-statistics-admin/src/services/publicationService.ts

```diff
@@ -94,6 +94,12 @@ export interface SavePublicationRequest {
   topicId: string;
 }
 
+export interface getReleasesParams {
```

Comment:
Types should be PascalCased i.e. `GetReleaseParams`. Also would consider naming this `ListReleasesParams`